ExampleFederation.ttl

```diff
@@ -6,7 +6,9 @@ PREFIX ex:     <http://example.org/>
 ex:dbpediaSPARQL
       a            fd:FederationMember ;
       fd:interface [ a                  fd:SPARQLEndpointInterface ;
-                     fd:endpointAddress <http://dbpedia.org/sparql> ] .
+                     fd:endpointAddress <http://dbpedia.org/sparql> ;
+                     fd:vocabularyMappings "dbpedia/vocabularyMappings.nt"
```

Comment:
Instead of making `fd:vocabularyMappings` a property of the interface of a federation member, I would make it a property of the federation member itself. In other words, for this particular case, `fd:vocabularyMappings` should be a property of `ex:dbpediaSPARQL`.

Additionally, I would name this property `fd:vocabularyMappingsFile` because, instead of connecting the fed.member to the actual vocab.mapping, it connects the fed.member to the file that describes the vocab.mapping.